ui/app/models/totp-key.js

```diff
@@ -101,26 +135,31 @@ export default class TotpKeyModel extends Model {
   skew;
 
   @attr('boolean', {
+    editType: 'toggleButton',
     defaultValue: true,
+    helperTextDisabled: 'Vault will not return QR code and url upon key creation.',
+    helperTextEnabled: 'QR code and URL will be returned upon generating a key.',
   })
   exported;
 
-  // Doesn't really make sense as we can generate our own QR code from the url
   @attr('number', {
+    label: 'QR size',
     defaultValue: 0,
   })
-  qr_size;
+  qrSize;
 
   // Used when generate is false
   @attr('string', {
     label: 'URL',
     helpText:
       'If a URL is provided the other fields can be left empty. E.g. otpauth://totp/Vault:test@test.com?secret=Y64VEVMBTSXCYIWRSHRNDZW62MPGVU2G&issuer=Vault',
+    subText: `The TOTP key url string that can be used to configure a key.`,
   })
   url;
 
   @attr('string', {
-    label: 'Shared master key',
+    label: 'Key',
+    subText: `The root key used to generate a TOTP code.`,
```

Comment:
nit: [form field](https://github.com/hashicorp/vault/blob/main/ui/lib/core/addon/components/form-field.js#L132-L141) automatically capitalizes the attribute name, so unless we need to change that behavior (like above, we want the label to read `URL` and not `Url`), there's no need to pass a "label" key
```suggestion
    subText: 'The root key used to generate a TOTP code.',
```